packages/@react-aria/radio/docs/useRadioGroup.mdx

```diff
@@ -23,7 +23,7 @@ import {useRadioGroup, useRadio} from '@react-aria/radio';
 
 ---
 category: Forms
-keywords: [radiobutton, aria, form]
+keywords: [radiobutton, radio, input, aria, form]
```

Comment:
since category is added, does it make sense to have forms and form?

Reply:
sure I can remove form